cryptographic_estimators/MAYOEstimator/MAYOAlgorithms/direct_attack.py

```diff
@@ -78,6 +80,29 @@ def __init__(self, problem: MAYOProblem, **kwargs):
         
         self._fastest_algorithm = None
 
+    @optimal_parameter
+    def K(self):
+        """
+        Return the optimal parameter `K` from Furue, Nakamura, and Takagi strategy
```

Comment:
This is not clear to me. The constructor says it is a direct attack via the MQ Estimator. Why is there an additional optimal parameter K? If this is a different strategy to obtain a MQ instance from MAYO, shouldn't this be an extra attack? Also there should be a citation towards Furue, Nakamura, and Takagi

Reply:
I agree on adding a citation to the paper of Furue, Nakamura, and Takagi.

Yes, it is a different strategy that haven't yet been implemented In the MQEstimator.

Reply:
It shouldn't be an extra attack since it is how the direct_attack is defined in the MAYO specs.